repo_builder.sh

```diff
@@ -1,27 +1,22 @@
 #!/bin/bash
 
-############################################
-# This script will install and configure:  #
-# - rbenv                                  #
-# - hyku/hyrax                                 #
-############################################
-
-echo SETTING LOCAL VARIABLES
-
-export GEM_KEY=hull_culture
-export GEM_SOURCE=https://github.com/research-technologies
-export APPLICATION_KEY=hull_culture
-export APP_URL='https://github.com/research-technologies/hyrax_leaf'
-export BRANCH='master'
+while getopts gemkey:gemsrc: option; do
+  case ${option} in
+  gemkey) GEM_KEY=${OPTARG};;
+  gemsrc) GEM_SOURCE=${OPTARG};;
+  esac
+done
 
```

Comment:
If you set these env variables in .env, wouldn't they just be available to this file without having to pass the args and read them?

Reply:
oh, yes ... and I thought I was being so clever with my params!